front/app/components/DescriptionBuilder/Editor/Editor.tsx

```diff
@@ -7,7 +7,7 @@ import {
   Resolver,
 } from '@craftjs/core';
 
-import RenderNode from './RenderNode';
+import RenderNode from 'containers/Admin/pagesAndMenu/containers/ContentBuilder/components/Editor/RenderNode';
 
```

Comment:
@jamesspeake Only other difference I saw when I checked locally between the two render nodes was how they approach `isChildOfComplexComponent`.

Might be worth a quick check there as well to make sure the behaviour doesn't break :)